setup.py

```diff
@@ -34,7 +34,7 @@ def extract_metaitem(meta):
     maintainer = extract_metaitem('author'),
     maintainer_email = extract_metaitem('email'),
     packages = find_packages(exclude = ('tests', 'doc')),
-    install_requires = ['requests', 'requests-toolbelt'],
+    install_requires = ['requests', 'requests-toolbelt', 'jwcrypto', 'python-jose', 'django<2'],
```

Comment:
Could you use URLValidator to avoid adding django dependency
https://github.com/adonisnafeh/urlvalidator